renku/cli/_graph.py

```diff
@@ -279,10 +279,7 @@ def process_dependencies(self, dependencies, visited=None):
             visited.add(processing.commit)
 
             # Do the node processing here:
-            activity = Activity.from_git_commit(
-                processing.commit,
-                client=processing.client,
-            )
+            activity = processing.client.process_commit(processing.commit)
```

Comment:
Since we have a comment on a process_commit method, we can remove the above comment.